engine/server/engine/main.go

```diff
@@ -134,8 +134,8 @@ func runMain() error {
 		return stacktrace.Propagate(err, "Failed to create an enclave manager for backend type '%v' and config '%+v'", serverArgs.KurtosisBackendType, backendConfig)
 	}
 
+	// TODO: replace with persistent client so that we can get logs even after enclave is stopped.
```

Comment:
Still not understanding what this means

Reply:
I believe it means that the `logsDatabaseClient` should implement another client, one that calls the logs storage instead of the `KurtosisBackend` impl. which uses Docker and K8s clients under the hood. Am I right @tedim52?

Reply:
You are correct leo